frontend/src/components/PaymentGateway/PaymentGateway.tsx

```diff
@@ -17,18 +18,27 @@ export const SubscriptionPlans: React.FC<{ stripePromise: Promise<Stripe | null>
 
     const [plans, setPlans] = useState<any[]>([]);
     const [currentPlan, setCurrentPlan] = useState(organization.subscriptionId ? 1 : 0);
-
+    const [prices, setPrices] = useState<any[]>([]);
+    const [error, setError] = useState<string | null>(null);
+    
     useEffect(() => {
-        setPlans([
-            {
-                id: "price_1PYvHVEpF6ccgZLwn6uq6d4J",
-                name: "Enterprise Plan",
-                description: "Access to all features including premium support.",
-                price: "30.00",
-                interval: "month"
+        // Fetch product prices when the component mounts
+        async function fetchPrices() {
+            try {
+                const data = await getProductPrices({ user, product_id:"prod_R05WPWPAgXt6Kj" });
```

Comment:
@ramirezmorac2 this should be saved in an environment var `product_id:"prod_R05WPWPAgXt6Kj"`

Reply:
Ok, I'm going to fix it now